tests/features/ExampleTest.php

```diff
@@ -3,13 +3,12 @@
 
 class ExampleTest extends FeaturesTestCase
 {
-
     /**
      * A basic functional test example.
      *
      * @return void
      */
-    function test_basic_example()
+    public function test_basic_example()
```

Comment:
__[code_sniffer]__
* Method name "ExampleTest::test_basic_example" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->